assets/src/design-system/components/contextMenu/animatedContextMenu.js

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
Most of this file has been copied over from https://github.com/google/web-stories-wp/blob/main/assets/src/dashboard/components/popoverMenu/popoverCard.js.

I've marked the spots with new code